### PR TITLE
Monorepo (Turbo Repo)

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

This is a big one. This will allow us to:

- Keep code modularized and organized
- Reuse as much logic as possible
- Establish local pipelines (build needs to happen before test, etc.)
- Take advantage of Vercel's turbo caching
- And many more benefits

NOTE: Before merging we need to [update the build output directory in Vercel](https://vercel.com/docs/concepts/git/monorepos#step-2:-set-the-build-command) (That's why is failing right now)

refs #1581

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How should this be tested?

- [ ] Be sure to move your `.env` file to `apps/web/.env`
- [ ] Download changes and run
- [ ] `yarn dx`
- [ ] `yarn dev`
- [ ] `yarn build`
- [ ] `yarn start` (after build)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code and corrected any misspellings
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]
after moving the .env into  everything worked

**Comment by zomars** [comment]
> after moving the .env into everything worked

Added to testing steps 👍🏽

**Review by edanfesi — APPROVED** [review]

File: apps/web/next.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
-const withTM = require("@vercel/edge-functions-ui/transpile")(["react-timezone-select"]);
+const withTM = require("@vercel/edge-functions-ui/transpile")([]);
```
Comment (emrysal) [inline apps/web/next.config.js:2]: We no longer need to transpile react-timezone-select? In that case we can probably remove the entire HOC

**Review by emrysal — APPROVED** [review]

**Review by jamiepine — APPROVED** [review]
Very exited for this, can't wait to dig into organizing shared components and logic. It all works on my end with no issues, incredible job @zomars!

**Comment by denik1981** [comment]
@zomars Nice work! Turbo is amazing :100: 

I feel that `.env.example` got lost in the transition. 
Is that possible?

**Comment by zomars** [comment]
> @zomars Nice work! Turbo is amazing 💯
> 
> I feel that `.env.example` got lost in the transition. Is that possible?

It's on it's new home at ´apps/web/.env.example´

**Pull request merged**